### PR TITLE
Add tasks to process and clean up triggers

### PR DESCRIPTION
**Pull request by kvsn**

### CODE REVIEW

File: djtriggers/models.py [hunk 1]
```diff
@@ -57,7 +59,7 @@ class Trigger(models.Model):
 
     trigger_type = models.CharField(max_length=50, db_index=True)
     source = models.CharField(max_length=250, null=True, blank=True)
-    date_received = models.DateTimeField()
+    date_received = models.DateTimeField(default=datetime.now)
```
Comment (pieterdd) [inline djtriggers/models.py]: I think you want this one instead: https://docs.djangoproject.com/en/1.7/ref/models/fields/#django.db.models.DateField.auto_now_add

`datetime.now` is - as far as I can tell - a method object.

**Comment by pieterdd** [comment]
As far as I understand your code, you:

- Extracted the logic for trigger cleaning and processing
- Created two `@shared_task`s (these will not automatically be run by Celery, correct?)
- Initialize `date_received` to the current time by default 

I have left one comment to investigate. Also, do the tests pass when running them locally? I'd recommend that you add a [`circle.yml` file](https://circleci.com/docs/configuration) for `django-triggers` so that they can be run automatically.

File: djtriggers/models.py (hunk 1 above)
Comment (kvsn) [inline djtriggers/models.py]: I prefer `datetime.now`, since the `auto_now` options cannot be overridden.

Comment (pieterdd) [inline djtriggers/models.py]: Are you sure that it's not `datetime.now()`?

**Comment by kvsn** [comment]
Indeed.

**Comment by pieterdd** [comment]
:shipit:

**Pull request merged**